LoopBackTests/server/package.json

```diff
@@ -4,8 +4,9 @@
   "description": "A test server for running loopback-clients integration testing.",
   "main": "index.js",
   "dependencies": {
-    "loopback": "1.x.x",
-    "loopback-component-push": "~1.2.2",
-    "loopback-component-storage": "~1.0.5"
+    "loopback": "^2.14.0",
+    "loopback-component-push": "^1.x",
+    "loopback-component-storage": "^1.x",
```

Comment:
Nitpick: use `1.x` or `^1.2.3`, replacing `1.2.3` with the latest published version. The latter form is IMO the best.